src/plugins/table/schema.js

```diff
@@ -0,0 +1,182 @@
+// Copyright 2018 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+function normalizeTable(change, error) {
+    const { code: violation, child } = error
+
+    switch (violation) {
+        case 'child_type_invalid':
+            console.log('child_type_invalid')
+            console.log('child.type:', child.type)
+            if (child.type === 'title') {
+                change.setNodeByKey(child.key, { type: 'caption' })
+            } else {
+                console.warn('Unhandled table violation:', violation, JSON.stringify(error, null, 2))
+            }
+            break
+
+        default:
+            console.warn('Unhandled table violation:', violation)
+            break
+    }
+}
+
+function normalizeTgroup(change, error) {
+    console.warn('Unhandled tgroup violation:', error.code)
+}
+
+function normalizeColspec(change, error) {
+    console.warn('Unhandled colspec violation:', error.code)
+}
+
+function normalizeSpanspec(change, error) {
+    console.warn('Unhandled spanspec violation:', error.code)
+}
+
+function normalizeThead(change, error) {
+    console.warn('Unhandled thead violation:', error.code)
+}
+
+function normalizeTbody(change, error) {
+    console.warn('Unhandled tbody violation:', error.code)
+}
+
+function normalizeTfoot(change, error) {
+    console.warn('Unhandled tfoot violation:', error.code)
+}
+
+function normalizeRow(change, error) {
+    console.warn('Unhandled row violation:', error.code)
+}
+
+function normalizeEntry(change, error) {
+    console.warn('Unhandled entry violation:', error.code)
+}
+
+function normalizeCaption(change, error) {
+    console.warn('Unhandled caption violation:', error.code)
+}
+
+export default {
+    blocks: {
+        table: {
+            data: {
+                summary: s => s == null || typeof s === 'string',
+                frame: f => f == null || typeof f === 'string',
+                colsep: c => c == null || c >= 0,
+                rowsep: r => r == null || r >= 0,
+                pgwide: p => p == null || p >= 0,
+            },
```

Comment:
Even though it's a temporary solution we might still want a proper schema for those attributes.